sktime/forecasting/boxcox_bias_adjusted_forecaster.py

```diff
@@ -0,0 +1,191 @@
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+
+"""BoxCoxBiasAdjustedForecaster implementation."""
+
+__author__ = ["sanskarmodi8"]
+
+import numpy as np
+import pandas as pd
+
+from sktime.forecasting.base import BaseForecaster
+from sktime.transformations.series.boxcox import BoxCoxTransformer
+
+
+class BoxCoxBiasAdjustedForecaster(BaseForecaster):
+    """Box-Cox Bias-Adjusted Forecaster.
+
+    This module implements a forecaster that applies Box-Cox transformation
+    and bias adjustment to the predictions of a wrapped forecaster.
+
+    The bias adjustment is implemented using Taylor series expansion of the
+    method described in:
+    Forecasting: Principles and Practice (2nd ed)
+    Rob J Hyndman and George Athanasopoulos
+    Monash University, Australia. OTexts.com/fpp2.
+
+    For methods like `predict_proba`, the behavior of the wrapped forecaster
+    is directly used without any additional adjustments. This means that
+    probability estimates are provided as they are by the underlying forecaster.
+    Users should ensure that the wrapped forecaster's `predict_proba` output
+    is suitable for their needs, as no transformation or adjustment will be applied
+    to these estimates by the `BoxCoxBiasAdjustedForecaster`.
+
+    Parameters
+    ----------
+    forecaster : BaseForecaster
+        The wrapped forecaster to which Box-Cox transformation and
+        bias adjustment will be applied.
+    lmbda : float, optional (default=None)
+        The Box-Cox transformation parameter. If None, it will be estimated.
+    """
+
+    def __init__(self, forecaster, lmbda=None):
+        self.forecaster = forecaster
+        self.lmbda = lmbda
+        self.boxcox_transformer_ = None
+        # Clone tags from the wrapped forecaster to this forecaster
+        self._set_delegated_tags(self.forecaster)
```

Comment:
as it does not inherit from the `_DelegatedForecaster`, this will not work. Perhaps it makes sense to move the method out and call it from both places?

Reply:
Should I do something like below?
```python
class BoxCoxBiasAdjustedForecaster(BaseForecaster):
    """Box-Cox Bias-Adjusted Forecaster.

    This module implements a forecaster that applies Box-Cox transformation
    and bias adjustment to the predictions of a wrapped forecaster.

    The bias adjustment is implemented using Taylor series expansion of the
    method described in:
    Forecasting: Principles and Practice (2nd ed)
    Rob J Hyndman and George Athanasopoulos
    Monash University, Australia. OTexts.com/fpp2.

    Parameters
    ----------
    forecaster : BaseForecaster
        The wrapped forecaster to which Box-Cox transformation and
        bias adjustment will be applied.
    lmbda : float, optional (default=None)
        The Box-Cox transformation parameter. If None, it will be estimated.
    """

    def __init__(self, forecaster, lmbda=None):
        self.forecaster = forecaster
        self.lmbda = lmbda
        self.boxcox_transformer_ = None

        # Clone tags from the wrapped forecaster to this forecaster
        self._clone_tags_from_forecaster(forecaster)
        super().__init__()

    def _clone_tags_from_forecaster(self, forecaster):
        """Clone tags from the wrapped forecaster to this forecaster.

        Parameters
        ----------
        forecaster : BaseForecaster
            The forecaster from which to clone tags.
        """
        tags = forecaster.get_tags()
        for tag_name, tag_value in tags.items():
            self.set_tag(tag_name, tag_value)

```

Reply:
the method already exists as `clone_tags`, the problem is we do no twant to clone all the tags, such as author fields or internal conversion tags - only the tags that make sense to clone. These are currently cloned by the delegator method, but this does not exist as a loose method unfortunately.